MdeModulePkg/Library/VarCheckHiiLib/VarCheckHiiGen.c

```diff
@@ -1,14 +1,19 @@
 /** @file
   Var Check Hii bin generation.
 
-Copyright (c) 2015 - 2018, Intel Corporation. All rights reserved.<BR>
+Copyright (c) 2015 - 2024, Intel Corporation. All rights reserved.<BR>
 SPDX-License-Identifier: BSD-2-Clause-Patent
 
 **/
 
 #include "VarCheckHiiGen.h"
+#include "VarCheckHii.h"
 
-LIST_ENTRY  mVarCheckHiiList = INITIALIZE_LIST_HEAD_VARIABLE (mVarCheckHiiList);
+VAR_CHECK_HII_VARIABLE_HEADER  *mVarCheckHiiBin    = NULL;
+UINTN                          mVarCheckHiiBinSize = 0;
+LIST_ENTRY                     mVarCheckHiiList    = INITIALIZE_LIST_HEAD_VARIABLE (mVarCheckHiiList);
+
+EFI_GUID  gVarCheckReceivedHiiBinHandlerGuid = VAR_CHECK_RECEIVED_HII_BIN_HANDLER_GUID;

```

Comment:
should move to VarCheckHii.h:

extern EFI_GUID  gVarCheckReceivedHiiBinHandlerGuid = VAR_CHECK_RECEIVED_HII_BIN_HANDLER_GUID;